homeassistant/components/zwave_js/device_action.py

```diff
@@ -186,8 +186,9 @@ async def async_get_actions(
         # underlying value is not being monitored by HA so we shouldn't allow
         # actions against it.
         if (
-            state := hass.states.get(entry.entity_id)
-        ) and state.state == STATE_UNAVAILABLE:
+            not (state := hass.states.get(entry.entity_id))
```

Comment:
This is not about test coverage, it's about handling the case where the entity has been removed?

Reply:
this is actually a bug fix because if the state is not found, we should just continue and that was not handled properly. The test coverage piece is the change below to reduce missed lines. Renamed the PR accordingly